<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDlg</class>
 <widget class="QDialog" name="SearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="grpKey">
      <property name="title">
       <string>Key</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lneKey"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chkKeyCase">
         <property name="text">
          <string>Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdKeyExactMatch">
         <property name="text">
          <string>Exact match</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdKeyWildcards">
         <property name="text">
          <string>Wildcards</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdKeyRegexp">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpValue">
      <property name="title">
       <string>Value</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lneValue"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chkValueCase">
         <property name="text">
          <string>Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdValueExactMatch">
         <property name="text">
          <string>Exact match</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdValueWildcards">
         <property name="text">
          <string>Wildcards</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdValueRegexp">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdValueHex">
         <property name="text">
          <string>Hex</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdgReplace" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Replace value with</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneReplace"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkHex">
        <property name="text">
         <string>hex</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Direction</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rdUp">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdDown">
          <property name="text">
           <string>Down</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QPushButton" name="btnReplaceAll">
         <property name="text">
          <string>Replace All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnSearchNext">
         <property name="text">
          <string>Find Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="lblItemsFound">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnReplace">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearchNext</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>searchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grpKey</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>updateSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grpValue</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>updateSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneKey</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchDlg</receiver>
   <slot>updateSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchDlg</receiver>
   <slot>updateSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneKey</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grpKey</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKeyCase</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdKeyExactMatch</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdKeyWildcards</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdKeyRegexp</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grpValue</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkValueCase</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdValueExactMatch</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdValueWildcards</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdValueRegexp</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdValueHex</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>resetSearchList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReplaceAll</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>replaceAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReplace</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>replace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchNext()</slot>
  <slot>updateSearchNext()</slot>
  <slot>resetSearchList()</slot>
  <slot>replace()</slot>
  <slot>replaceAll()</slot>
 </slots>
</ui>
